plugins/headless/hu.bme.mit.gamma.headless.server/target/classes/config.properties

```diff
@@ -0,0 +1,5 @@
+%headless.gamma.path=E:/Egyetem/GammaWrapper/HeadlessEclipse/gammaapi/eclipse
+headless.gamma.path=D:/2-Programming/Work/BME2021/MUNKA/openapi_linux_theta/eclipse/eclipse
+%headless.generator.path=E:/Egyetem/GammaWrapper/HeadlessEclipse/generator/eclipse.exe
+headless.generator.path=D:/2-Programming/Work/BME2021/MUNKA/openapi_linux_theta/generator/eclipse
+root.of.workspaces.path=D:/2-Programming/Work/BME2021/MUNKA/theta_workspaces/
```

Comment:
`plugins/headless/hu.bme.mit.gamma.headless.server/target` is not added in gitignore